schemas/computation.schema.tpl.json

```diff
@@ -0,0 +1,84 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/Computation",
+    "_extends": "../../core/research/protocolExecution.schema.tpl.json",
+    "required": [
+        "input",
+        "output",
+        "protocol",
```

Comment:
You could remove those since they are already required fields for the protocolExecution schema itself.